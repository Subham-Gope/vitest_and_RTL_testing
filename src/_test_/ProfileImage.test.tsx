// import { describe, test, expect } from "vitest";
// import { screen, render } from "@testing-library/react";
// import ProfileImage from "../components/ProfileImage";

// describe("testing the ProfileImage.tsx file", () => {
//   test("testing if the img tag has the alt title of 'profile pic", () => {
//     render(<ProfileImage />);

//     const img = screen.getByAltText("profile pic");
//     expect(img).toBeInTheDocument();
//   });
// });
