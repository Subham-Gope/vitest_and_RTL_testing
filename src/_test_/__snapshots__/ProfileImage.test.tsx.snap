// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing the ProfileImage.tsx file > testing if the img tag has the alt title of 'profile pic 1`] = `
<div>
  <img
    alt="profile pic"
    src="https://www.pexels.com/photo/elegant-museum-interior-with-classical-sculpture-29844114/"
  />
</div>
`;
